backend/server/schema.py

```diff
@@ -27,8 +35,18 @@ class MatchType(DjangoObjectType):
     class Meta:
         model = Match
 
-    winner = graphene.Field(TeamType, id=graphene.Int(), name=graphene.String())
+    winner = graphene.Field(TeamType)
     year = graphene.Int()
+    home_team = graphene.Field(TeamType)
+    away_team = graphene.Field(TeamType)
+
+    @staticmethod
```

Comment:
@cfranklin11 I understand that static methods are those that can be invoked without a class instantiation....are these methods @staticmethod for that reason?

Reply:
Yeah, per the `graphene` documentation, all resolvers are static methods, and the first argument is always the `root`, or underlying object, not `self`, which would be an instance of the `ObjectType` class